tests/integration/conftest.py

```diff
@@ -311,7 +334,8 @@ def pytest_runtest_makereport(item, call):
         extra.append(pytest_html.extras.json(arguments, name="Test arguments"))
 
         # Extra files to be added in 'Links' section
-        for filepath in (LOG_FILE_PATH, WAZUH_CONF):
+        files = get_report_files()
+        for filepath in files:
```

Comment:
When I add a file that not exists the all execution fail. 
I executed: 
 `python3 -m pytest -svx test_vulnerability_detector/test_feeds/ --html=/vagrant/r1_test.html --add-file=/var/ossec/logs/archives/archives.json > /vagrant/LogsConFails.log`
I Attach results [Logs.zip](https://github.com/wazuh/wazuh-qa/files/7086637/SampleOfFileNotExists.zip). 

For this case, it also should work without problems. (only the file will do not attached to the log)